SwitchSpeak-iOS/CrumbManagerTest/CrumbManagerTest.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  CrumbManagerTest.swift
+//  CrumbManagerTest
+//
+//  Created by Ning Luo on 2018/3/25.
+//  Copyright © 2018年 SwitchSpeak. All rights reserved.
+//
+
+import XCTest
+@testable import SwitchSpeak_iOS
+
+class CrumbManagerTest: XCTestCase {
+    var CrumbManagerTest:CrumbStack!
+    override func setUp() {
+        CrumbManagerTest = CrumbStack()
+        super.setUp()
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        CrumbManagerTest=nil
+        super.tearDown()
+    }
+    
+    func emptyCrumstacktest() {
```

Comment:
*`emptyCrumbstackTest`
Don't forget the 'b' in 'crumb' and please put this in camelCase.